src/Servers/HttpSys/src/PublicAPI.Unshipped.txt

```diff
@@ -36,3 +36,7 @@ Microsoft.AspNetCore.Server.HttpSys.IHttpSysRequestTimingFeature
 Microsoft.AspNetCore.Server.HttpSys.IHttpSysRequestTimingFeature.Timestamps.get -> System.ReadOnlySpan<long>
 Microsoft.AspNetCore.Server.HttpSys.IHttpSysRequestTimingFeature.TryGetElapsedTime(Microsoft.AspNetCore.Server.HttpSys.HttpSysRequestTimingType startingTimestampType, Microsoft.AspNetCore.Server.HttpSys.HttpSysRequestTimingType endingTimestampType, out System.TimeSpan elapsed) -> bool
 Microsoft.AspNetCore.Server.HttpSys.IHttpSysRequestTimingFeature.TryGetTimestamp(Microsoft.AspNetCore.Server.HttpSys.HttpSysRequestTimingType timestampType, out long timestamp) -> bool
+Microsoft.AspNetCore.Server.HttpSys.AuthenticationManager.CaptureCredential.get -> bool
```

Comment:
As @Tratcher pointed out, we'll need an API [issue](https://github.com/dotnet/aspnetcore/issues/new?assignees=&labels=api-suggestion&projects=&template=30_api_proposal.md&title=) for this.  Please link to this PR.

Reply:
Will do.

Reply:
> As @Tratcher pointed out, we'll need an API [issue](https://github.com/dotnet/aspnetcore/issues/new?assignees=&labels=api-suggestion&projects=&template=30_api_proposal.md&title=) for this. Please link to this PR.

I created an API proposal: https://github.com/dotnet/aspnetcore/issues/51990

(For some reason it didn't get labeled as an `(api-suggestion)`, although I think I used an appropriate template. Could you  help me with adjusting that?)